include/triton/common/error.h

```diff
@@ -26,6 +26,7 @@
 #pragma once
 
 #include <string>
+#include <cstdint>
```

Comment:
Tested on Fedora 40 with gcc version 14.1.1 and it is indeed necessary to add this inclusion for compilation.